### DISCUSSION TITLE
Insights Single Metric

### DISCUSSION THREAD

**Discussion by connorwinston** [directus/directus#15714] [Archive]
I am trying to get the sum of the balance of our invoices, but I keep getting a GraphQL Error and want to double-check I'm not configuring it wrong.

Insights Panel:
![image](https://user-images.githubusercontent.com/12714889/191828968-825e48a2-bfe2-4735-961a-6f4eef0637a2.png)

Field Settings:
![image](https://user-images.githubusercontent.com/12714889/191829134-0ae09cb2-85a6-49c4-bbcc-206fdb597991.png)

GraphQL Error:
```json
{
    "errors": [
        {
            "message": "GraphQL validation error.",
            "extensions": {
                "code": "GRAPHQL_VALIDATION_EXCEPTION",
                "graphqlErrors": [
                    {
                        "message": "Cannot query field \"balance\" on type \"invoices_aggregated_fields\".",
                        "locations": [
                            {
                                "line": 1,
                                "column": 1029
                            }
                        ],
                        "path": [
                            "query_16e61c0a"
                        ]
                    }
                ]
            }
        }
    ]
}
```

GraphQL Query:
```json
{"query":"query { query_16e61c0a: invoices_aggregated (limit: 1, sort: \"balance\", filter: {_and: [{status: {_in: [\"pending\", \"quote\", \"paid\"]}}]}) { sum { balance } } }"}
```

**Comment by azrikahar** [comment]
This might be the same as #15663 as both are using aggregation on bigInteger field in GraphQL. More info: https://github.com/directus/directus/issues/15663#issuecomment-1253164852

**Reply by connorwinston** [reply]
Yep, this was it. Converting them to regular Integers did the trick!
